templates/try-matrix.html

```diff
@@ -24,14 +27,16 @@ <h3>Choose a client</h3>
         <div class="step choose-a-server">
             <div class="instructions">
                 <h3>Choose a server</h3>
-                <p>Matrix.org is provided for free by the Foundation, but we
-                    encourage you to <a href="https://joinmatrix.org/servers">find an existing server</a>
-                    or get hosting for your organisation.
+                <p>
+                    Matrix is a decentralised protocol: like email, you need to have an account provider to start using
+                    matrix.
+                </p>
+                <p>
+                    But picking a provider can be overwhelming for your first steps. The Matrix.org Foundation is
+                    hosting
+                    the matrix.org Matrix server for every newcomer to create an account. You can safely use it to
```

Comment:
I think it would be nice to say how to use it. I think thats one of the key issues of the blogpost we saw. We dont talk enough about the "how".

Reply:
Hm, do you suggest adding instructions regarding how to create an account using Element?

Reply:
If we link to element directly now, I would argue we should do that yes.

Reply:
Does it work for you if we link to https://matrix.org/docs/chat_basics/matrix-for-im/#creating-a-matrix-account ?

Reply:
yes